src/tap/src/tapcell.i

```diff
@@ -37,11 +37,86 @@
 
 #include "tap/tapcell.h"
 #include "ord/OpenRoad.hh"
+#include "opendb/db.h"
 
-tap::Tapcell *
-getTapcell()
+namespace ord {
+
+tap::Tapcell* getTapcell();
+
+}
+
+using ord::getTapcell;
+using std::vector;
+using std::set;
+using std::string;
+
+%}
+
+
+%include "../../Exception.i"
+
+%inline %{
+
+namespace tap {
+
+void run(odb::dbMaster* endcap_master, int halo_x, int halo_y, const char* cnrcap_nwin_master, const char* cnrcap_nwout_master, const char* endcap_prefix, int add_boundary_cell, const char* tap_nwintie_master, const char* tap_nwin2_master, const char* tap_nwin3_master, const char* tap_nwouttie_master, const char* tap_nwout2_master, const char* tap_nwout3_master, const char* incnrcap_nwin_master, const char* incnrcap_nwout_master, const char* tapcell_master, int dist, const char* tap_prefix)
```

Comment:
"run" is too gen

Reply:
I reformatted with clang-format now, thank you!